app/models/project.rb

```diff
@@ -43,6 +43,18 @@ class Project < ActiveRecord::Base
 
   scope :search, ->(name) { where("name like ?", "%#{name}%") }
 
+  def template
```

Comment:
can make this a has_one -> get caching from rails ... also name it `template_stage` ?